detect.py

```diff
@@ -88,7 +90,9 @@ def detect(weights='yolov7.pt',
             p, s, im0, frame = path, '', im0s, getattr(dataset, 'frame', 0)
 
             p = Path(p)  # to Path
-            save_path = str(save_dir / p.name)  # img.jpg
+            filename = p.name.replace(" ", "_")
```

Comment:
extract these three lines to a function?

Hot keys: (Select the three lines) - CTRL + ALT + M 

``` 
save_path, txt_path = method_name(p, save_dir, save_txt)

def method_name(p, save_dir, save_txt):
    filename = p.name.replace(" ", "_")
    save_path = str(save_dir / filename)  # img.jpg
    txt_path = str(save_txt / (filename.split('.')[0] + '.txt'))
    return save_path, txt_path
```

Reply:
@michaelalassaad  refactor the method to the following and make sure that it does not break any functionalities